test/scenarios/bookshop/strictMode.test.js

```diff
@@ -0,0 +1,128 @@
+const cds = require('../../cds.js')
```

Comment:
This test file is still not being executed. As it is not included inside the `test/scenarios/bookshop/index.js` file.

Reply:
yes i moved it to sqlite/general so we can test this feature flag separately